components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -14,868 +14,982 @@ import * as React from 'react';
 import * as request from 'request';
 
 import Gapi from './Gapi';
-import { encryptPassword, flattenDeploymentOperationError, log, wait } from './Utils';
+import {encryptPassword, flattenDeploymentOperationError, getFileContentsFromGitHub, log, wait} from './Utils';
 
-// TODO(jlewi): Can we fetch these directly from GitHub so we always get the latest value?
-// When I tried using fetch API to do that I ran into errors that I interpreted as chrome blocking
-// downloads from other domains from the one the app is being served at.
-// So for security reasons it might be better to just bundle the configs.
-// When we build a docker image as part of our release process we can just
-// copy in the latest configs.
-import appConfigPath from './user_config/app-config.yaml';
+/** Relative paths from the root of the repository. */
+enum ConfigPath {
+    V05 = 'components/gcp-click-to-deploy/app-config.yaml',
+    V06 = 'bootstrap/config/kfctl_gcp_iap.0.6.yaml'
```

Comment:
Shall we just load different config template for iap vs basic-auth cases?

Reply:
Good point. I think it might be easier to remove the IAP-specific sections from this config if the user chooses not to use IAP rather than have to manually add them if basing the config on the non-IAP file.